erdjs/src/dapp/walletProvider.ts

```diff
@@ -138,6 +139,49 @@ export class WalletProvider implements IDappProvider {
         });
     }
 
+     /**
+     * Fetches the logout hook url and redirects the client to the wallet logout.
+     */
+    async logout(): Promise<boolean> {
+        if (!this.mainFrame) {
+            return false;
+        }
+
+        const {contentWindow} = this.mainFrame;
+        if (!contentWindow) {
+            console.warn("Something went wrong, main wallet iframe does not contain a contentWindow");
```

Comment:
maybe start the log with a lowercase character like in the rest of the file?

Reply:
done